kotsadm/web/src/components/PreflightResultPage.jsx

```diff
@@ -106,7 +111,9 @@ class PreflightResultPage extends Component {
         this.props.data.refetch();
       })
       .catch((err) => {
-        console.log(err);
+        this.setState({
+          errorMessage: err ? `Encountered an error while trying to ignore permissions: ${err.message}` : "Something went wrong, please try again."
```

Comment:
same here, we should reset `errorMessage` to `""` at the beginning of the function